dev-java/zxing/zxing-3.5.1.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+JAVA_PKG_IUSE="doc source test"
+MAVEN_ID="com.google.zxing:core:3.5.2"
+JAVA_TESTING_FRAMEWORKS="junit-4"
+
+inherit java-pkg-2 java-pkg-simple
+
+DESCRIPTION="Core barcode encoding/decoding library"
+HOMEPAGE="https://github.com/zxing/zxing/core"
+SRC_URI="https://github.com/zxing/zxing/archive/${P}.tar.gz"
+LICENSE="Apache-2.0"
+SLOT="3"
+KEYWORDS="~amd64 ~x86"
+
+CP_DEPEND="dev-java/jcommander:0"
+DEPEND="
+	${CP_DEPEND}
+	>=virtual/jdk-1.8:*
+"
+RDEPEND="
+	${CP_DEPEND}
+	>=virtual/jre-1.8:*
+"
+
+S="${WORKDIR}/${PN}-${P}"
+
+JAVA_SRC_DIR=( {core,javase}/src/main/java )
```

Comment:
This puts `javase` and `core` in one single jar file. 
Upstream has them separated:
```
./javase/target/javase-3.5.1-sources.jar
./javase/target/javase-3.5.1-javadoc.jar
./javase/target/javase-3.5.1.jar
./core/target/core-3.5.1-sources.jar
./core/target/core-3.5.1-javadoc.jar
./core/target/core-3.5.1.jar
```
When separating ([example](https://gitweb.gentoo.org/repo/gentoo.git/tree/dev-java/jdom/jdom-2.0.6.1.ebuild))  don't forget:
Missing: https://github.com/zxing/zxing/blob/zxing-3.5.1/core/pom.xml#L82
Missing https://github.com/zxing/zxing/blob/zxing-3.5.1/javase/pom.xml#L61

Reply:
splitted, it indeed simplify the ebuild